kuma/users/apps.py

```diff
@@ -0,0 +1,110 @@
+from django.conf import settings
+from django.contrib import messages
+from django.contrib.auth.apps import AuthConfig
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models import signals
+from django.utils.translation import ugettext_lazy as _
+
+from allauth.account.signals import user_signed_up, email_confirmed
+from allauth.socialaccount.signals import social_account_removed
+from waffle import switch_is_active
+
+from kuma.core.urlresolvers import reverse
+
+from .jobs import UserGravatarURLJob
+from .tasks import send_welcome_email
+
+
+class UserConfig(AuthConfig):
+    """
+    The Django App Config class to store information about the users app
+    and do startup time things.
+    """
+    name = 'kuma.users'
+    verbose_name = _('User')
+
+    def ready(self):
+        super(UserConfig, self).ready()
+
+        # the user signal handlers to connect to
+        User = self.get_model('User')
+        signals.post_save.connect(self.on_user_save,
+                                  sender=User,
+                                  dispatch_uid='users.user.post_save')
+        user_signed_up.connect(self.on_user_signed_up)
+        email_confirmed.connect(self.on_email_confirmed)
+        social_account_removed.connect(self.on_social_account_removed)
+
+        # the user ban signal handlers to connect to
+        UserBan = self.get_model('UserBan')
+        signals.post_delete.connect(self.on_ban_delete,
+                                    sender=UserBan,
+                                    dispatch_uid='users.user_ban.delete')
```

Comment:
I don't quite remember but you're right, I just moved this into the apps module as a signal callback, too.